dataactcore/utils/jobQueue.py

```diff
@@ -28,11 +29,13 @@ def __init__(self, job_queue_url="localhost"):
         @self.jobQueue.task(name='jobQueue.enqueue')
         def enqueue(jobID):
             # Don't need to worry about the response currently
+            CloudLogger.log("Adding job {} to the queue".format(str(jobID)))
```

Comment:
I'm not fully up to speed on the job manager, so asking as a general question--is there any other info that would be useful to log here? (`queueURL`, etc.)

Paging @nmonga91, who would have better insight into this.